router.go

```
@@ -375,6 +390,47 @@ func (app *App) register(methods []string, pathRaw string, group *Group, handler
 	}
 }
 
+func (app *App) routeExists(method string, pathRaw string) bool {
```

golangci-lint (1.62.2)

File is not `gofumpt`-ed with `-extra`

(gofumpt)